src/Model/Core/BunqModel.php

```diff
@@ -349,6 +354,21 @@ protected static function fromJson(BunqResponseRaw $responseRaw, string $wrapper
         return new BunqResponse($value[self::INDEX_FIRST], $responseRaw->getHeaders());
     }
 
+    /**
+     * @param $path
+     *
+     * @return BunqModel
+     * @throws BunqException
```

Comment:
this function doesnt throw an exception. we dont bubble up exceptions in docblock